### PR TITLE
Add strong password note to passphrase encryption

### PR DESCRIPTION
**Pull request by IkerAlus**
In the pages _forging.adoc_ and _cryptography.adoc_

### CODE REVIEW

**Review by Tschakki — CHANGES_REQUESTED** [review]
Maybe it would help to describe what is considered a strong password in this context. Or Add a link to a page, which is covering the topic.

**Comment by IkerAlus** [comment]
> Maybe it would help to describe what is considered a strong password in this context. Or Add a link to a page, which is covering the topic.

I was planning to do so and link to [this wiki article](https://en.wikipedia.org/wiki/Password_strength) which has a very comprehensive description of the concept. But I did not see any external link in these pages so I thought maybe we prefer not to include those. If you think it is ok to have an external here I will add the mentioned link as suggested.

**Comment by Tschakki** [comment]
Hmm the article is very general. It does not give me an answer what is secure in this regard. I think we should give concrete guidelines here to avoid confusion. For example if we expect to use the 12 word mnemonic for the password as well, we should mention it directly. We can still include this external link for reference. Generally, there are no places in docs where it's forbidden to add external links :)

**Comment by IkerAlus** [comment]
The point here is that this is not a basic user level application where we can set strict rules, but a tool aimed to delegates and node operators. There should be not limitations or basic guidelines. If we wanted to be precise and complete regarding the password strength concept we would need to elaborate like in the wiki article, which is not the point of these doc pages. Moreover, that article has the precise guidelines for what we understand as a _strong password_ in the section "Guidelines for strong passwords". What we can do is reference to that exact section to avoid the reader to get lost in the bigger wiki article.

File: modules/ROOT/pages/guides/node-management/forging.adoc [hunk 1]
```diff
@@ -61,6 +61,11 @@ Please re-enter your password: ***
 <1> Enter the secret passphrase here that needs to be encrypted.
 <2> Enter the password here that will be required to decrypt the passphrase again.
 
+[NOTE]
+====
+Be sure to use a link:https://en.wikipedia.org/wiki/Password_strength#Guidelines_for_strong_passwords[strong password].
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/node-management/forging.adoc]: Please always use link attributes to insert links:
On top of the page under settings:

```
:url_wikipedia_password_strength: https://en.wikipedia.org/wiki/Password_strength#Guidelines_for_strong_passwords
```

Then use the attribute in the text.
```
Be sure to use a strong password.

See the {url_wikipedia_password_strength}[Guidelines for password strength^] at Wikipedia for reference. 
```

The caret at the end will open the link in a new tab.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/guides/node-management/forging.adoc (hunk 1 above)
Comment (IkerAlus) [reply modules/ROOT/pages/guides/node-management/forging.adoc]: added, thanks!

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**